package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@snap/camera-kit-react-native",
-  "version": "0.1.2",
+  "version": "0.1.3",
```

Comment:
so the package version will be `0.1.3` in npm? in the npm world community follows `major.minor.patch` convention (semver):
1. The major part is updated when there are backwards incompatible changes introduced (think of consumers of their package, when they pull it will that break compilation their app).
2. The minor part is updated when features are added and there are not breaking changes.
3. The patch is updated when it it is a bugfix

Given that in our case we update to the LTS version of Camera Kit, I would update the major or minor part.

Reply:
Let's do 0.2.0 then